### ISSUE TITLE
Clarify accessibility summary language in the three documents

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#476]
From the German working group looking at the guidelines, we learned that our documents could make it more clear. 

Manfred wrote:
The accessibility summary was intended (in EPUB Accessibility 1.0) to describe in human-readable prose the accessibility features present in the publication as well as any shortcomings. From EPUB Accessibility version 1.1 the accessibility summary became a human-readable summary of the accessibility that complements, but does not duplicate, the other discoverability metadata.

From EPUB Techniques: https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/techniques/epub-metadata/
Still old definition:
accessibility_summary
If true it indicates that the accessibilitySummary is present in the package document, otherwise if false it means that the metadata is not present.
This means there is a human-written text containing a short explanatory summary of the accessibility of the product or the URL of a web page comprising such a summary. Summarizes the already existent information and may add information that the publisher could not express with the other codes.

From ONIX Techniques: https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/techniques/onix-metadata/
Still old definition:
accessibility_summary
Returns the description of code 00 of codelist 196 (Accessibility summary) if present in the ONIX record, otherwise if false it means that the metadata is not present.
This means there is a human-written text containing a short explanatory summary of the accessibility of the product or the URL of a web page comprising such a summary. Summarizes the already existent information and may add information that the publisher could not express with the other codes.

George comments:

While technically the information is aligned, I can see where it may be confusing. We just need to make it more clear.

I suggest we change the word "from" to "starting with" In the guidelines. In the techniques, we can say:
This means there is a human-readable summary of the accessibility that complements, but does not duplicate, the other discoverability metadata, or there is a  URL of a web page comprising such a summary.

**Comment by chrisONIX** [comment]
For ONIX techniques - the code 00 is for the Accessibility Summary as defined in EPUB 1.0 - we can not change how it is defined as this breaks backward compatibility 
So for the accessibility summary from EPUB 1.1 we have code 92 - Accessibility Addendum which is referenced 
https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/techniques/onix-metadata/#understanding-the-variables-6
accessibility_addendum - do we need to make the difference clearer?

**Comment by GeorgeKerscher** [comment]
We should make it clear that 00 is depricated and 92 is what is current.

When it comes to the techniques, if 92 is present , then we display that.
If 00 is present, we display that.
If neither are present, we display no accessibility summary information is present.

How does that sound?

**Comment by ManfredMuchenberger** [comment]
The prevailing opinion in the Börsenverein working group is that, according to the new definition of "accessibility summary", **only new information on accessibility and barriers** may be included and not information that is already contained in other accessibility metadata.
Is this interpretation correct?

Perhaps this is a language problem.
The way I read it, the wording ‘**Summarises the already existing information**’ in
https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/techniques/epub-metadata/
and
https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/techniques/onix-metadata/
contradicts the Guidelines
https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/guidelines/
which states ‘**but does not duplicate**’.

**Comment by GeorgeKerscher** [comment]
Yes, that is correct. The accessibility summary adds additional information that cannot be obtained by the other accessibility metadata.

We will attempt to make that more clear in the techniques based on information in this thread. This new approach to accessibility metadata is aligned with Accessibility Metadata Display Guide for Digital Publications 2.0.

The older approach was duplicating information, and we want to get away from that approach.

**Comment by mattgarrish** [comment]
> The accessibility summary adds additional information that cannot be obtained by the other accessibility metadata.

Right, otherwise the summary would potentially be a redundant duplication of everything these guides recommend generating. And the user would still be stuck reading through it again to find out if it presents any new information they need to be aware of.
